README.md

```diff
@@ -66,7 +66,6 @@ Add or update any `wp_nav_menu()` functions in your theme (often found in `heade
 ```php
 wp_nav_menu( array(
 	'theme_location'  => 'primary',
-	'depth'	          => 2, // 1 = no dropdowns, 2 = with dropdowns.
```

Comment:
Instead of removing the `depth` here could you instead comment it out? 

I've had a fair few people open up support issues before the inline comment showing which depth values do what because they were unclear what values to use. This comment solves that.

Maybe comment out the line and add another comment above explaining why it's not needed?

Reply:
I think this is a wordpress related question.
wp-bootstrap-navwalker is just a replacement for the default nav walker and must behave as wp documentation says.

Anyway, I've modified the line hoping it is clear.